### PR TITLE
ES2015ify and remove support for Node.js versions older than v4

### PR DESCRIPTION
**Pull request by kevva**
Depends on https://github.com/gruntjs/grunt-contrib-imagemin/pull/378.

### CODE REVIEW

**Comment by sindresorhus** [comment]
@kevva Can you rebase?

**Comment by kevva** [comment]
Done.

File: tasks/imagemin.js [hunk 1]
```diff
@@ -1,82 +1,83 @@
...
-			});
-		}, function (err) {
-			if (err) {
-				grunt.warn(err);
+				grunt.file.write(file.dest, optimizedBuf);
+				grunt.verbose.writeln(chalk.green('✔ ') + file.src[0] + chalk.gray(` (${msg})`));
+			})
+			.catch(err => {
+				grunt.warn(`${err} in file ${file.src[0]}`);
+			}), {
```
Comment (sindresorhus) [inline tasks/imagemin.js]: The code here is a bit hard to read. Can you extract line 49 to here out into a separate variable?

**Comment by jorrit** [comment]
@sindresorhus and @kevva: thanks for your renewed attention to this module. I am wondering if you are going to take a look at the issue queue as well? Some of the issues will be fixed with your changes, but some might not.

**Comment by kevva** [comment]
@jorrit, yes. I will when it's merged.

**Comment by jorrit** [comment]
Thanks!

File: tasks/imagemin.js (hunk 1 above)
Comment (kevva) [inline tasks/imagemin.js]: Yeah, I agree.

**Comment by sindresorhus** [comment]
Awesome 💃

**Pull request merged**